lib/resin-lint.coffee

```diff
@@ -8,14 +10,26 @@ reporter = require('coffeelint/lib/reporters/default')
 
 CONFIG_PATH = path.join(__dirname, '../config/coffeelint.json')
 
+# The linter expects the path to actual source files, for example:
+#		src/
+#		test/
+#	but depcheck expects the root of a project directory (where the
+#	package.json is). This function takes a path and propagates upwards
+#	until it contains a package.json
+getPackageJsonDir = (dir) ->
```

Comment:
I wanted the interface to be exactly the same, so the linting still goes on as well, and all subsequent calls will still work, just if you add an extra `-u` it will *also* check deps.